app/controllers/rhapsody_realities.js

```diff
@@ -70,8 +70,24 @@ class RhapsodyController {
     * @param res
     */
     list(req, res) {
+        let viewData = {};
+
+        let params = {};
+        if (req.query.page) {
+            params.page = req.query.page;
+        }
+
+        this.rhapsodyService.listRhapsodies(params)
+            .then(data => {
+                
+                viewData.rhapsodies = data.rhapsodies;
+                viewData.pagination = data.pagination;
+                res.render('admin_list_rhapsody', { viewData, layout: 'admin_main' });
+            })
+            .catch(err => {
+                throw err;
```

Comment:
hmm, not entirely sure what will happen here tbh
i would have expected that we still rendered the view, but pass the error as well as empty rhapsody data so that user can get a finer view.


e.g `res.render('admin_list_rhapsody', { {}, layout: 'admin_main' });`